scientific/src/commonMain/kotlin/unit/DimensionlessUnit.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ Copyright 2022 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.scientific.unit
+
+import com.splendo.kaluga.base.utils.Decimal
+import com.splendo.kaluga.scientific.PhysicalQuantity
+import kotlinx.serialization.Serializable
+
+@Serializable
+sealed class Dimensionless : ScientificUnit<PhysicalQuantity.Dimensionless>,
+    MetricAndImperialScientificUnit<PhysicalQuantity.Dimensionless>
+
+object Percent : Dimensionless(), MetricBaseUnit<MeasurementSystem.MetricAndImperial, PhysicalQuantity.Dimensionless> {
+    override val symbol: String = "%"
+    override val system = MeasurementSystem.MetricAndImperial
+    override val quantity = PhysicalQuantity.Dimensionless
+    override fun fromSIUnit(value: Decimal): Decimal = value
```

Comment:
I would argue we need a converter from a double to a percentage. By that I mean that if I want 10 percent, I should declare a Scientific unit of 10(Percent). But usually in programming, we will use 0.1 to represent 10 percent. So its probably best to declare the SI unit as 1.0 == 100 percent

Also, if we're adding percentage, we should also promilage